unix/vncserver/vncserver.in

```diff
@@ -35,7 +35,15 @@
 # your site
 #
 
+# Start with legacy ~/.vnc user dir
 $vncUserDir = "$ENV{HOME}/.vnc";
+if (stat($vncUserDir)) {
+  warn "~/.vnc is deprecated, please migrate to XDGBDS-compliant paths!";
```

Comment:
I'm not sure "XDGBDS" is a well known abbreviation. I would suggest something a bit more explicit.

Reply:
By the end of this PR I would hope to improve that message even further by linking to, say, a migration guide in the OP of the PR.